content/python/concepts/keywords/terms/lambda/lambda.md

```diff
@@ -0,0 +1,44 @@
+---
+Title: 'Lambda'
```

Comment:
This is redundant with Anonymous Functions. If it's included, maybe we just want a link back to that entry.

Reply:
I can link out to the Anonymous Functions entry but I think it could be worth keeping this since `lambda` is a reserved Python keyword.